### ISSUE TITLE
[diagnostic] Pareto view of the v1.4 board (chart + script, not a score change)

### ISSUE THREAD

**Issue by coezbek** [fstandhartinger/jevbench#64]
The v1.4 README and RESULTS show one axis at a time as bars. I found it useful to see Intelligence
against the cost axis on one plane, with the Pareto frontier drawn, so here is that view built
purely from `results/v1.4/jevbench-v1.4-results.json` (board of 2026-09-23, 71 ranked entries).
Nothing in it is measured outside JevBench.

<img width="1600" height="1067" alt="Image" src="https://github.com/user-attachments/assets/c4e2e46b-7176-44cc-9e63-ee346f6eea4c" />

How to read it:

- x is the board's cost per 1,000 decisions (log scale), y is v1.4 Intelligence. The stepped line
  is the Pareto frontier over the ranked entries: nothing sits above-left of it.
- Colour is the serving footprint: open models by the VRAM their board configuration needs
  (under 8, 16 or 24 GB), hollow circles for entries without published weights. Footprint is my
  bucketing from the `underlying` and `licence` fields plus the model sizes; happy to correct any.
- Labels are shortened; the numbered key maps them back to the board's display names.

Three things the picture shows that the bar charts don't:

1. The frontier splits in two. Its cheap half is entirely open weights (Certo, Laya, jeff,
   system-one-open, raw Qwen3-4B logits, reflex 4B: six of the ten frontier points), and it tops
   out at Intelligence 47.5 with reflex 4B, six points under Jev at half Jev's cost. Above that the
   frontier is closed: JevK5, Jev, then nothing until GPT-6 Luna.
2. The band between Intelligence 55 and 90 is almost empty: djev (thinking) at 72 is the only
   entry in it. Everything else is either a Jev-class model at 45 to 53 or a frontier LLM at 93+.
3. Between $0.05 and $0.25 per 1,000 decisions the board holds most of the open 8B to 35B
   models, and none of them beats reflex 4B on Intelligence. On this board, spending more than
   about $0.04 per 1,000 decisions buys nothing until the jump to a frontier LLM.

Reasons we made this, because we wanted to find a model at the ultra-cheap edge. We think Laya currently is positioned well there.

We put up an endpoint to curl against it using the same format as Cloudflare's Jev API. Free, no key:

    curl https://laya.inference.zaitlabs.com/ai/run \
      -H "Content-Type: application/json" \
      -d '{
        "state": "Help! My payouts have been failing for 3 days.",
        "questions": {
          "is_urgent": {"type": "noul", "instructions": "Does this convey urgency?"}
        }
      }'

    {"model":"laya-0.3.4/typed-decisions",
     "answers":{"is_urgent":{"type":"noul","noul":0.7894}},
     "usage":{"input_tokens":43,"output_tokens":0}}

Requests and responses may be stored and used to tLink header on
every response say so), so don't send anything sensitive. Hosted in the EU, docs at https://laya.inference.zaitlabs.com/docs

**Comment by fstandhartinger** [comment]
Thanks — Harold has logged this for review. Contributor text and code are treated as untrusted data; no external code was run on the benchmark host.

- Harold
